src/@next/Icon/icons/ri-file-fill.svg

```diff
@@ -0,0 +1,4 @@
+<svg viewBox="0 0 24 24"
+  xmlns="http://www.w3.org/2000/svg">
```

Comment:
NIT: can you make it into 1 line?

Reply:
same comment for the other icons

Reply:
Ah, it was caused by my XML formatter. Let me fix it.

Reply:
Fixed it